stat.go

```diff
@@ -79,7 +79,10 @@ func (fi *FileInfo) Mode() os.FileMode {
 }
 
 func (fi *FileInfo) ModTime() time.Time {
-	return time.Unix(int64(fi.status.GetModificationTime())/1000, 0)
+	mtime := fi.status.GetModificationTime()
+	sec := mtime / 1000
+	msec := mtime % 1000
+	return time.Unix(int64(sec), int64(msec*1e6))
```

Comment:
I think this can be `time.Unix(0, fi.status.GetModificationTime() * int64(time.Millisecond))`

Reply:
Oh! That's even better.